### ISSUE TITLE
Enum values not sanitized anymore

### ISSUE THREAD

**Issue by maartencardinaels** [anymaniax/orval#510]
Since the latest version (6.9.1), enum values are [not sanitized anymore](https://github.com/anymaniax/orval/commit/47182342ea50f28233371819614388c3beb797a1?diff=split#diff-308070d35b5355e7127225498bc272877a211f174b4f5d946325a7c2f1f73f7a). This means that 

```
....
enum:
   [
      'key1/key2/key3',
      'key4/key5/key6,
   ]
...
```

now is converted into:

```
export const SomeDdto = {
  'key1/key2/key3': 'key1/key2/key3',
  'key4/key5/key6': 'key4/key5/key6',
}
```

while before this resulted into:

```
export const SomeDdto = {
  key1key2key3: 'key1/key2/key3',
  key4key5key6: 'key4/key5/key6',
}
```
and we are using `SomeDdto.key1key2key3`

I'm not sure if this should be considered a bug - and if there is valid reasoning behind this change, we need to adapt our code that is using the generated DTOs.

**Comment by anymaniax** [comment]
It's a bug

**Comment by maartencardinaels** [comment]
@anymaniax, ok thanks for confirming already

**Comment by anymaniax** [comment]
Should be fixed with 6.9.2 and sorry for that
